build-config/scripts/onie-mk-itb.sh

```diff
@@ -139,6 +141,7 @@ DTB="$(realpath $DTB)"
 			arch = "$ARCH";
 			os = "linux";
 			compression = "none";
+			load = <$FDT_LOAD>;
```

Comment:
I looked at the `dtc` compiler -- it does not care about the `load` parameter.

Thinking more the real question is what does u-boot do when loading an .itb where the `dtb` block does not specify the `load` variable.  Meaning what has u-boot been doing on `powerpc` and `armv7` when loading an .itb?